geanydoc/src/Makefile.am

```diff
@@ -7,7 +7,7 @@ geanydoc_la_SOURCES = \
 	geanydoc.c \
 	geanydoc.h
 
-geanydoc_la_CFLAGS = $(AM_CFLAGS)
+geanydoc_la_CPPFLAGS = $(AM_CPPFLAGS) -DG_LOG_DOMAIN=\"GeanyDoc\"
```

Comment:
`AM_CFLAGS` is the default for when there is no traget-specific `CFLAGS`, so yeah should be fine.